rfc/rfc003-oauth2-authorization.md

```diff
@@ -293,15 +302,29 @@ GET  /fhir/Patient/1 HTTP/1.1
 
 A token MAY be used multiple times unless the returned error code prevents it. A token MUST NOT be used when it has been expired.
 
-### 6.2. Authorization
+### 6.2 Authorization
 
 The resource server MUST validate the validity of the access token. It MAY contact the authorization server to validate the token or it MAY use existing knowledge to validate the token. For example a JWT can be validated by using the registered public key of the authorization server. The resource server MUST also check if the client certificate used for the TLS connections is from the same party that requested the access token. The next step is to validate if the token may be used to access the requested resource. There are three different cases that MUST be supported:
 
 1. **The requested resource does not contain patient information.** Certain resources do not contain patient information and may therefore be exchanged without user context. Resources that fall in this category MUST be marked as such in the specific use case specification.
 2. **The requested resource belongs to a patient.** In this case the resource server MUST validate that user context is present, e.g. an access token has been requested with the _usi_ field. The resource server MUST also verify a Nuts Authorization Credential was used in the access token request for the combination of custodian, actor, subject and resource.
 3. **The actor and custodian are the same.** It may be the case that a care organization is using multiple service providers. In that case each service provider acts on behalf of the care organization. Therefore, it's not needed to provide user context. It's up to the service providers to provide the correct enforcement of roles and any auditing duties. Each of the service providers \(actor and custodian\) MAY use different identifiers for the same care organization.
 
-### 6.3. Error codes
+In the first two cases, the resource server MUST check if the policy covers the requested resource.
+
+### 6.3 Error codes
 
 Different protocols return different types of error messages. The format will most likely also differ. This means that error messages have to be specified per use-case. If an error message supports a text-based error code, then it should support the illegal\_access\_token code. If a client receives this error code then it MUST NOT reuse the access token.
 
+## 7. Bolt requirements
+
+Different types of data require different levels of authorization.
+Because those requirements depend on the data, it's impossible for an RFC to specify these.
+This passes the requirement on to the Bolts.
+A Bolt MUST add a chapter defining the access policy.
+The policy MUST define which resources may be accessed when no restrictions are given.
+The resources MAY be seperated into 3 categories:
+
+- Personal: personal and/or medical resources.
```

Comment:
this is non-normative, just a suggestion on how to structure the Bolt access policy?

Reply:
yes, that's why it says MAY